src/main/java/uk/gov/ch/controller/update/ManagingOfficerDataController.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.ch.controller.update;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.ManagingOfficerCountNotFoundException;
+import uk.gov.ch.model.update.ManagingOfficerData;
+import uk.gov.ch.service.update.ManagingOfficerDataService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import java.util.List;
+
+@RestController
+public class ManagingOfficerDataController {
+
+    @Autowired
+    private ManagingOfficerDataService managingOfficerService;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OracleQueryApplication.APPLICATION_NAME_SPACE);
+
+    @GetMapping("/managing-officer/{incorporationNumber}/data")
+    public ResponseEntity<List<ManagingOfficerData>> getManagingOfficerData(
+            @PathVariable String incorporationNumber,
+            @RequestParam(name = "start_index", defaultValue = "0", required = false) int startIndex,
+            @RequestParam(name = "items_per_page", defaultValue = "15", required = false) int itemsPerPage) {
+
+        LOGGER.info("Calling service to retrieve active officers' details for company number " + incorporationNumber);
```

Comment:
Maybe change this to use string interpolation for consistency- like the below examples

Reply:
fixed